mediacloud/mediawords/util/html.py

```diff
@@ -1,3 +1,6 @@
+"""Various utility functions for handling html."""
+
+from bs4 import BeautifulSoup
```

Comment:
super nit: `pip` installed packages go after system packages, and before local packages (so `BeautifulSoup` goes in its own group after `typing`)

Reply:
fixed.